### PR TITLE
add some diagnostic logging

### PR DESCRIPTION
**Pull request by devoncarew**
add some diagnostic logging:

- log process start and stop to the IntelliJ log file
- add a user toggleable option to enable more verbose debug logging
- when that's set, start flutter run apps up with `--verbose`; this can help identify a lot of issues with the various toolchains (android, gradle, ios, ...) (the same as `flutter run -v` on the cli)

screen shot 2017-05-19 at 4 05 07 pm

@skybrian @pq

### CODE REVIEW

File: src/io/flutter/run/SdkRunConfig.java [hunk 1]
```diff
@@ -43,6 +44,8 @@
  */
 public class SdkRunConfig extends LocatableConfigurationBase
   implements LaunchState.RunConfig, RefactoringListenerProvider, RunConfigurationWithSuppressedDefaultRunAction {
+  private static final Logger LOG = Logger.getInstance(SdkRunConfig.class);
```
Comment (skybrian) [inline src/io/flutter/run/SdkRunConfig.java]: I don't see where this is used.

File: src/io/flutter/run/daemon/FlutterApp.java [hunk 2]
```diff
@@ -119,7 +119,7 @@ public static FlutterApp start(Project project, @NotNull RunMode mode,
                                  @NotNull String analyticsStart,
                                  @NotNull String analyticsStop)
     throws ExecutionException {
-
+    LOG.info(command.toString());
```
Comment (skybrian) [inline src/io/flutter/run/daemon/FlutterApp.java]: Add project name and mode here too?

Comment (devoncarew) [inline src/io/flutter/run/daemon/FlutterApp.java]: done

File: src/io/flutter/run/SdkRunConfig.java (hunk 1 above)
Comment (devoncarew) [inline src/io/flutter/run/SdkRunConfig.java]: removed

**Pull request merged**